Traceback (most recent call last):
  File "badly.py", line 5, in <module>
    inverse(0)
  File "badly.py", line 3, in inverse
    return 1/x
ZeroDivisionError: division by zero
